Dashboard/load.py

```diff
@@ -26,20 +26,24 @@ def insert_product_information(conn: connection, extracted_data: dict):
 `url`, `product_name`, `product_code`, `website_id`.')
 
     logging.info("Inserting product data into the database")
-    with get_cursor(conn) as cur:
-        cur.execute(
-            """INSERT INTO PRODUCTS (website_id, url, product_code, product_name)
-            VALUES (%s, %s, %s, %s)""",
-            (extracted_data['website_id'], extracted_data['url'],
-             extracted_data['product_code'], extracted_data['product_name']))
-        conn.commit()
-    logging.info("Product information successfully added into the database")
-
-
-def load_product_data(config: _Environ, product_url: str) -> None:
+    try:
+        with get_cursor(conn) as cur:
+            cur.execute(
+                """INSERT INTO PRODUCTS (website_id, url, product_code, product_name)
+                VALUES (%s, %s, %s, %s)""",
+                (extracted_data['website_id'], extracted_data['url'],
+                extracted_data['product_code'], extracted_data['product_name']))
+            conn.commit()
+            logging.info("Product information successfully added into the database")
+            return True
+    except Exception:
```

Comment:
Use a more specific error

Reply:
This is client facing, so using a general exception must be used to stop clients getting exceptions!